nfs-ganesha/build/build_deb

```diff
@@ -146,7 +146,7 @@ EOF
     $VENV/chacractl repo update ${chacra_endpoint}
```

Comment:
You need to add ``/flavors/${FLAVOR}`` here as well.  Actually, I think with debian you can just keep the flavors bit of the url in ``chacra_endpoint``.